Cargo.toml

```diff
@@ -5,3 +5,8 @@ members = [
   "tracing-actix-web-mozlog",
   "actix-web-location",
 ]
+
+[workspace.package]
+version = "0.8.0"
```

Comment:
I don't think you should do workspace lib versioning like this. Recommend keeping each crate with it's own version field. Other shared fields below here make sense though.

Reply:
I go back and forth about this quite a bit. The plus being when we tag a release, do a library check, etc. we only have one location we need to update. 
The downside is that these crates are very loosely bound together under a fairly arbitrary collector. 

I'll revert the version number change for now, but I still kinda would like to keep the version work simplified where possible.